sycl/test/on-device/basic_tests/buffer/buffer_create.cpp

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: gpu,level_zero
```

Comment:
this test needs to be E2E and go into llvm-test-suite

Reply:
Yes, that is the plan. It is here so it can be reviewed.
Later, after the plugin change has been merged, the test will be moved to llvm-test-suite.

Reply:
OK

Reply:
I am closing this PR since with no plugin change required, the test can go to llvm-test-suite directly.